### ISSUE TITLE
Should we set CPU_FTR_UNALIGNED_LD_STD on P7/P8?

### ISSUE THREAD

**Issue by mpe** [linuxppc/linux#76]

**Comment by npiggin** [comment]
I would say yes, in that P7/8 don't have a big penalty for unaligned. But do we have POWER >= 7 overrides for copyuser and memcpy so it would effectively be a noop?

**Comment by mpe** [comment]
Yeah that might be true with the way the code is currently written. But we should still set it if it's correct on those CPUs.

Also the error cases in __copy_tofrom_user_power7() fall back to __copy_tofrom_user_base(), so we may hit it that way - although in that case performance is probably not our main concern.

I'll do a patch to turn it on and do some smoke tests.
